api/src/main/java/org/openmrs/notification/MessagePreparator.java

```diff
@@ -20,6 +20,6 @@
 	 * @param template
 	 * @return the prepared <code>Message</code>
 	 */
-	public Message prepare(Template template) throws MessageException;
+	//public Message prepare(Template template) throws MessageException;
```

Comment:
Yeah I actually went ahead and fixed a whole lot of stuff and it got wayyy to complex. So I'm doing it in parts. I'm currently dealing with this class now. It's a lot of files interlinked, espeically to this `MessagePreparator` class